packages/flutter/lib/src/material/app.dart

```diff
@@ -22,7 +22,19 @@ const TextStyle _errorTextStyle = const TextStyle(
   decorationStyle: TextDecorationStyle.double
 );
 
-
+/// An application that uses material design.
+///
+/// This widget is a convenience widget that wraps a number of widgets that are
```

Comment:
> This widget is a convenience widget that wraps

Suggestion:

A convenience widget that wraps a number of widgets that are commonly...
or 
A wrapper for a number of widgets that are commonly...